airgun/views/job_invocation.py

```diff
@@ -28,125 +27,89 @@ def is_displayed(self):
 
 class JobInvocationCreateView(BaseLoggedInView):
     breadcrumb = BreadCrumb()
-    job_category = FilteredDropdown(id='job_invocation_job_category')
-    job_template = FilteredDropdown(locator="//div[contains(@class, 'job_template_selector')]")
-    bookmark = FilteredDropdown(id='targeting_bookmark')
-    search_query = TextInput(name='targeting[search_query]')
-    template_content = ConditionalSwitchableView(reference='job_template')
-
-    @template_content.register('Run Command - SSH Default', default=True)
-    class RunSSHCommandForm(View):
-        command = TextInput(id='command')
-
-    @template_content.register('Power Action - SSH Default')
-    class RestartHostForm(View):
-        action = FilteredDropdown(id='s2id_action')
 
-    @template_content.register('Puppet Run Once - SSH Default')
-    class RunPuppetForm(View):
-        puppet_options = TextInput(id='puppet_options')
+    class category_and_template(WizardStepView):
+        expander = Text(".//button[contains(.,'Category and Template')]")
+        job_category = Select('OUIA-Generated-Select-single-1')
+        job_template = Select('OUIA-Generated-Select-typeahead-1')
 
-    @template_content.register('Module Action - SSH Default')
-    class RunModuleForm(View):
-        action = FilteredDropdown(id='s2id_action')
+    class target_hosts_and_inputs(WizardStepView):
+        expander = Text(".//button[contains(.,'Target hosts and inputs')]")
+        # command action
```

Comment:
In my opinion, these comments seem redundant and provide no added value. Instead, we could remove them and rely on the variable to determine the object type it represents.

Reply:
done, thanks